src/main/resources/hibernate.cfg.xml

```diff
@@ -4,7 +4,7 @@
 <hibernate-configuration>
 	<session-factory>
 		<property name="hibernate.hbm2ddl.auto">update</property>
-		<property name="hibernate.connection.release_mode">auto</property>
```

Comment:
Was ist mit Transaktionen? Warum soll nach jedem Statement die Connection (aggressiv) released werden? Eine Datenbanktransaktion benötigt immer die gleiche Connection, damit das funktioniert.
https://docs.jboss.org/hibernate/stable/core.old/reference/en/html/transactions-connection-release.html
"auto (the default) - this choice delegates to the release mode returned by the org.hibernate.transaction.TransactionFactory.getDefaultReleaseMode() method. For JTATransactionFactory, this returns ConnectionReleaseMode.AFTER_STATEMENT; for JDBCTransactionFactory, this returns ConnectionReleaseMode.AFTER_TRANSACTION. **It is rarely a good idea to change this default behavior as failures due to the value of this setting tend to indicate bugs and/or invalid assumptions in user code.**"

Reply:
Habe mit dem aktuellen Commits das Problem erstmal gelöst. Problem war ein Change von Kai.